src/ProfilesList.json

```
@@ -124,5 +124,6 @@
   "idevkshitij.json",
   "JesugG17.json",
   "akkash20.json",
+  "jarkyd.json",
```

```suggestion
  "JayRkyd.json",
```

Filename should be same.